README.md

```diff
@@ -14,7 +14,23 @@ LevelDOWN
 
 LevelDOWN was extracted from [LevelUP](https://github.com/level/levelup) and now serves as a stand-alone binding for LevelDB.
 
-It is **strongly recommended** that you use LevelUP in preference to LevelDOWN unless you have measurable performance reasons to do so. LevelUP is optimised for usability and safety. Although we are working to improve the safety of the LevelDOWN interface it is still easy to crash your Node process if you don't do things in just the right way.
+<a name="installation"></a>
+Installation
+----------------------------
+
+First you need to install [LevelUP](https://github.com/level/levelup)! With [npm](https://npmjs.org) do:
+
+```
+npm install levelup leveldown
```

Comment:
Note that this will not be enough, once we release `levelup@2.0.0`, since `levelup` no longer will default to `leveldown`.

Reply:
It will be enough, but you need to do `levelup(leveldown(location))` that is

Reply:
Hmm ok. Will `leveldown` / `abstract-leveldown` be compatible with both `levelup` 1.x and 2.x?

Reply:
(any changes planned?)

Reply:
> Hmm ok. Will leveldown / abstract-leveldown be compatible with both levelup 1.x and 2.x?

Yes.

Reply:
@vweevers The difference is that `levelup` no longer has encodings and will take a down instance as first parameter, instead of using `location` as it does now. And the Promise api of course.

Reply:
So maybe this (commit d609d131a1e2d6b7bf18b46b3ff346390ac1702f) was a mistake, seeing as it's not the whole story and levelup will be the better place to explain it. Shall I revert?

Reply:
Sure! I think also we should try to push using `level` as the first option and mention the `levelup/abstract-leveldown` combo as more advanced usage. `level` will bundle `levelup`, `encoding-down` and `leveldown`.

Reply:
Sounds like a plan!